### DISCUSSION TITLE
"npm start"  is ok, "yarn start" gives ESLINT error :  Definition for rule 'react-hooks/exhaustive-deps' was not found

### DISCUSSION THREAD

**Discussion by cansuk** [yarnpkg/berry#3419] [Q&A]
My React project is opening with npm start, when all the dependencies are installed with yarn(to different directory with a new package.json) it gives ESLINT error. 

> **Definition for rule 'react-hooks/exhaustive-deps' was not found** 

But these rule is disabled and added to .eslintrc config. 

npm start gives just warning but yarn start gives error.
Yarn does not see eslint config. 

.eslintrc :

![image](https://user-images.githubusercontent.com/30255893/132843015-54c02ec3-167f-4581-bd0f-308bc666bbab.png)

package.json start script : 

`"start": "rimraf ./build && react-scripts start",`

Let me know if further information needed.

**Comment by merceyz** [comment]
You shouldn't be mixing Yarn and npm, they are not compatible and behave differently

> Let me know if further information needed.

Enough to see what the problem is - https://stackoverflow.com/help/minimal-reproducible-example
